cloudstate/stateless_function_context.py

```diff
@@ -0,0 +1,18 @@
+import random
+import sys
+from dataclasses import dataclass, field
+from typing import List
+
+from cloudstate.contexts import ClientActionContext
+from cloudstate.entity_pb2 import Forward, SideEffect
+
+
+@dataclass
+class StatelessFunctionContext(ClientActionContext):
+    command_name: str
+    errors: List[str] = field(default_factory=list)
+    effects: List[SideEffect] = field(default_factory=list)
+    forward: Forward = None
+
+    # todo: is this correct? there is no command_id on the stateless function requests.
+    command_id = random.randint(0, sys.maxsize)
```

Comment:
Yes. As there is no state there is no need for the proxy to map an entity to a specific shard. So there is no need for a command_id